twitterFederation/services/comments/src/types/query.ts

```diff
@@ -0,0 +1,8 @@
+import { gql } from 'apollo-server';
+
+export default gql`
+  extend type Query {
+    comments: [Comment]
```

Comment:
Here I have a thought that may be worth to share with you. When there is a list, if you don't use an `!` for the type inside the list you're telling GraphQL that it can return something like this:
```
[ actualComment, null, null, anotherComment, null ]
```
which in most cases is not ideal.
Also I prefer to have the list as a non nullable field and use the empty list.